DESCRIPTION

```diff
@@ -0,0 +1,15 @@
+Package: dashCytoscape
+Title: A Component Library for Dash aimed at facilitating network visualization in Python, wrapped around Cytoscape.js
+Version: 0.1.1
+Authors @R: as.person(c(The Plotly Team <xinghan@plot.ly>))
```

Comment:
Should we have a cytoscape@plot.ly email too?

Reply:
Good idea! Will set that up.

Reply:
fixed in 1fa98d1